index.js

```diff
@@ -157,9 +206,28 @@ module.exports = {
           'southWestLatitude',
           'southWestLongitude',
           'strictBounds',
-          'countries'
+          'countries',
+          'splitAddress',
+          'addressParts',
+          'displaySplitAddress'
         ]
       }
     ].concat(options.arrangeFields || []);
+  },
+  construct (self, options) {
+    self.on('apostrophe-pages:beforeSend', 'addGoogleApiKey');
+    self.addGoogleApiKey = function (req) {
+      req.data.googleApiKey = options.googleApiKey;
+    };
+
+    self.sanitizeFormField = function (req, form, widget, input, output) {
+      if (widget.splitAddress && widget.addressParts && widget.addressParts.length) {
```

Comment:
Shouldn't we store both the split and the original input?

Reply:
Done